akka-http-core/src/test/scala/akka/http/javadsl/JavaInitializationSpec.scala

```diff
@@ -12,7 +12,7 @@ class JavaInitializationSpec extends WordSpec with Matchers {
 
     "initializes the right field" in {
       akka.http.scaladsl.model.headers.LanguageRange.`*` // first we touch the scala one, it should force init the Java one
-      akka.http.javadsl.model.headers.LanguageRange.ALL // touching this one should not fail
+      val _ = akka.http.javadsl.model.headers.LanguageRange.ALL // touching this one should not fail
```

Comment:
I guess scalac would panic that there's a pure statement here and it takes no effect?
Wouldn't we need to do the same for the other values?